### PR TITLE
[stable/mongodb] - Allow using custom initialization scripts

### PR DESCRIPTION
**Pull request by tompizmor**
Signed-off-by: tompizmor <tompizmor@gmail.com>

#### What this PR does / why we need it:
This PR adds a new feature to the chart. Now it is possible to execute your custom _.sh_, and _.js_ the first time the database image is initialized.

More info about the usage is documented in the README.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [X] Chart Version bumped

### CODE REVIEW

File: stable/mongodb/templates/deployment-standalone.yaml [hunk 1]
```diff
@@ -118,6 +120,9 @@ spec:
         resources:
 {{ toYaml .Values.resources | indent 10 }}
       volumes:
+      - name: custom-init-scripts
```
Comment (juan131) [inline stable/mongodb/templates/deployment-standalone.yaml:126]: Shouldn't this be optional? Only if: `(.Files.Glob "files/docker-entrypoint-initdb.d/*[sh|js]")`

**Review by juan131 — CHANGES_REQUESTED** [review]

File: stable/mongodb/templates/deployment-standalone.yaml (hunk 1 above)
Comment (tompizmor) [reply stable/mongodb/templates/deployment-standalone.yaml:126]: yes

**Review by juan131 — APPROVED** [review]
Great thanks!

**Pull request merged**